.config/forest.dic

```diff
@@ -30,3 +32,4 @@ tipset/SM
 tipsetkey/S
 tuple
 validator/S
+stringified
```

Comment:
Replace with 'pretty-printed'? Maybe use another wording?

> /// Print a stringified JSON-RPC error and exit

> /// Print a pretty-printed JSON-RPC error and exit

> /// Output a pretty-printed JSON-RPC error and exit

> /// Write a pretty-printed JSON-RPC error and exit

> /// Pretty-print a JSON-RPC error and exit

Reply:
Yea this is much better than throwing words in the dictionary